### ISSUE TITLE
Unexpected results evaluating if/else statements on latest master

### ISSUE THREAD

**Issue by fleupold** [JacobEberhardt/ZoKrates#106]
On latest master (https://github.com/JacobEberhardt/ZoKrates/commit/da5b13f845145cf43d555c7741158727ef0018a2) the following program fails:

```
def main():
  b = if 1 < 2 then 0 else 1 fi
  b == 0
  return 1
```
with

```
Condition not satisfied: b should equal 0, but was 3 0
```

I understand that there is still an open issue with `<` (https://github.com/JacobEberhardt/ZoKrates/issues/99), but I would at least expect `b` to be either `0` or `1` not `3`

Also checking out https://github.com/JacobEberhardt/ZoKrates/commit/3c36637521495d9632c8c5c524e790cf61081f6e the same example above works fine.

**Comment by Schaeff** [comment]
Hey @fleupold thanks for this, so latest master is `1babdff` so I guess you mean latest develop? I was able to reproduce and it seems related to another recent change on develop which is possibly unrelated to #100

**Comment by Schaeff** [comment]
Fixed in #107

**Comment by Schaeff** [comment]
Fix merged into develop, closing @fleupold
